spec/Cabin/Infotainment.vspec

```diff
@@ -267,4 +267,27 @@ SmartphoneProjection.SupportedMode:
   allowed: [ 'ANDROID_AUTO', 'APPLE_CARPLAY', 'MIRROR_LINK', 'OTHER' ]
   description: Supportable list for projection.
 
+SmartphoneScreenMirroring:
+  type: branch
+  description: All smartphone mirroring actions.
+
+SmartphoneScreenMirroring.Active:
+  datatype: string
+  type: actuator
+  allowed: [ 'NONE', 'ACTIVE', 'INACTIVE' ]
+  description: Mirroring activation info.
+  comment: NONE indicates that mirroring is not supported.
+
+SmartphoneScreenMirroring.Source:
+  datatype: string
+  type: actuator
+  allowed: ['USB', 'BLUETOOTH', 'WIFI']
+  description: Connectivity source selected for mirroring.
+
+SmartphoneScreenMirroring.Resolution:
+  datatype: string
+  type: attribute
```

Comment:
Is this the physical resolution offered by the HMI, i.e. static, or is it the current resolution as agreed by receiver (Infotainment) and sending device? 

One can in general discuss if the ScreenMirroring/SmartphoneProjection signals shall be sensor or actuators. I assume the actual setup/connection/pairing not will be managed by VSS so that VSS only is used for showing current status, and then all could be "sensors", but I have no problems calling them "actuators" either, in the end it all depends on implementation. But if this signal represents current resolution rather than static/constant display resolution the it should be "sensor" rather than "attribute".